fendermint/vm/interpreter/src/fvm/exec.rs

```diff
@@ -83,6 +83,36 @@ where
             anyhow::bail!("failed to apply block cron message: {}", err);
         }
 
+        // Push the current block hash to the chainmetadata actor
+        //
+        if let Some(block_hash) = state.block_hash() {
+            let params = fvm_ipld_encoding::RawBytes::serialize(
+                fendermint_actor_chainmetadata::PushBlockParams {
+                    epoch: height,
+                    block: block_hash,
+                },
+            )?;
+
+            let msg = FvmMessage {
+                from: system::SYSTEM_ACTOR_ADDR,
+                to: fvm_shared::address::Address::new_id(fendermint_actors::CHAINMETADATA_ACTOR_ID),
```

Comment:
```suggestion
                to: chainmetadata::CHAINMETADATA_ACTOR_ADDR,
```
Since it's a well known one it should conform to the established pattern IMO.